docs/supported_op_devices.py

```diff
@@ -28,14 +28,24 @@ def main(argv):
     doc_table += formater.format('', '', '', '', '', '', op_name_max_len = op_name_max_len, c='=')
     for op in sorted(all_ops, key=lambda v: str(v).lower()):
         schema = b.GetSchema(op)
+        op_full_name = op
+        if not op_full_name.startswith('_'):
+            op_full_name = op_full_name.replace('_', '.')
+        *submodule, op_name = op_full_name.split('.')
```

Comment:
I think that deserves a function in ops - it's used in two distant places and can go stale quite easily and errors can go unnoticed for a long time.